src/main/java/mate/academy/library/model/Book.java

```diff
@@ -1,27 +1,43 @@
-package com.example.library.model;
+package mate.academy.library.model;
 
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 import java.math.BigDecimal;
+import jakarta.persistence.Table;
 import lombok.Data;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Data
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id =?")
+@Where(clause = "is_deleted=false")
+@Table(name = "books")
 public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     @Column(nullable = false)
     private String title;
+
     @Column(nullable = false)
     private String author;
+
     @Column(nullable = false, unique = true)
     private String isbn;
+
     @Column(nullable = false)
     private BigDecimal price;
+
     private String description;
+    @Column(name = "cover_image")
     private String coverImage;

```

Comment:
```suggestion
    private String coverImage;

```
not consisstent with other fields